resolvers/projectResolver.ts

```diff
@@ -921,7 +921,30 @@ export class ProjectResolver {
   ): Promise<Boolean> {
     try {
       const user = await getLoggedInUser(ctx)
-      return await this.updateProjectStatus(projectId, ProjStatus.can, user)
+      const didDeactivate = await this.updateProjectStatus(projectId, ProjStatus.can, user)
+      if (didDeactivate)
+       {
+         const project = await Project.findOne({ id: projectId })
+         if (project)
+         {
+           const segmentProject = {
+             email: user.email,
+             title: project.title,
+             LastName: user.lastName,
+             FirstName: user.firstName,
+             OwnerId: project.admin,
+             slug: project.slug,
+           }
+
+      analytics.track(
+        'Project deactivated',
+        `givethId-${ctx.req.user.userId}`,
+        segmentProject,
+        null
+      )
+    }
+  }
```

Comment:
Looking good, add some spacing to align analytics.track method and the brackets underneath so that it's more readable. 
Align them with the bracket before the catch (error) method.